Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml

```diff
@@ -0,0 +1,55 @@
+- type: entity
+  name: random shotgun safe spawner
+  id: ShotgunSafeSpawner
```

Comment:
make a base gun safe spawner that has sprite + offset: 0 that they all copy pasted

also format with style guide i think we just use upstreams